src/imgtools/coretypes/masktypes/structureset.py

```
@@ -353,6 +353,16 @@
             msg = f"ContourSequence in ROI at index {roi_index} is missing 'ContourData'. "
             raise ROIContourError(msg)
+        elif (
+            geometric_type := roi_contour.ContourSequence[
+                0
+            ].ContourGeometricType
+        ) != "CLOSED_PLANAR":
+            msg = (
+                f"ContourSequence in ROI at index {roi_index} has unexpected "
+                f"ContourGeometricType: {geometric_type}. Expected 'CLOSED_PLANAR'."
+            )
+            raise ROIContourError(msg)
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "raise", remove the leading "el" from "elif"

(R1720)